src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatServiceBuilder.java

```diff
@@ -238,6 +268,19 @@ private static String validateHostname(String hostname) {
     }
 
     /**
+     * Sets a {@link Consumer} that performs additional configuration operations against
+     * the Tomcat {@link StandardServer} created by a {@link TomcatService}. This method can be invoked
+     * multiple times to add multiple {@code configurator}s.
+     */
+    public TomcatServiceBuilder configurator(Consumer<? super StandardServer> configurator) {
+        @SuppressWarnings("unchecked")
+        final Consumer<StandardServer> castConfigurator =
```

Comment:
The use of `super` is intentional because a configurator might expect `Server` or `LifeCycleMBean`. It seems like I can remove this unchecked cast simply by changing the type of `TomcatServiceConfig.configurators` to `List<? super StandardServer>`, so let me just fix it that way.